<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDialogCaseAdd</class>
 <widget class="QDialog" name="CDialogCaseAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建案件</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*QDialog  {
    background:rgb(229, 229, 229);
border-width:0px;
border-radius:5px;
}*/

QWidget
{
	font-size:15px;
	font-family:Microsoft Yahei;	
}
QDialog{background-color: rgb(100, 100, 100);
border-width:2px;
border-color:rgb(255,0,0);
}
QWidget{color:rgb(255, 255, 255)}

QLineEdit{color:rgb(0, 0, 0)}

QScrollBar:vertical {
    background: #000000;
    width: 16px;
    margin: 16px 0 16px 0;
}

QScrollBar::handle:vertical {
    background:rgb(229, 229, 229);
    min-height: 16px;
border-width:0px;
border-radius:5px;
/*	border-image: url(:/Skin/handle.png);*/
}

QScrollBar::add-line:vertical {
    background: #444444;
    height: 16px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical {
    background: #444444;
    height: 16px;
    subcontrol-position: top;
    subcontrol-origin: margin;

}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}

QScrollBar:up-arrow:vertical {
    border: 2px solid #333333;
    width: 6px;
    height: 6px;
    background: #5f5f5f;
	border-image:url(:/Skin/up.png)
}

QScrollBar:down-arrow:vertical {
    border: 2px solid #333333;
    width: 6px;
    height: 6px;
    background: #5f5f5f;
	border-image:url(:/Skin/down.png)
}


</string>
  </property>
  <widget class="QFrame" name="frameSysBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
	border-image:url(:/Skin/bg-above2.png);
}
QLabel
{
border-image:null;
color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="3">
       <widget class="QFrame" name="frame_sysmenu_2">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>122</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame
{border-image:null;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <zorder>BtnClose_2</zorder>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="logo_2">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/Skin/logo.png)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelTitle_2">
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>170</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#labelTitle
{
	color: rgb(255, 255, 255);
	font-size:18px;
	font-family:Microsoft Yahei;	
}</string>
        </property>
        <property name="text">
         <string>新建案件</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="BtnClose_2">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip">
         <string>关闭</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	
	border-image: url(:/Skin/close.png)0 0 repeat

}
QPushButton:hover
{	
	border-image: url(:/Skin/close.png)30 0 repeat

}
QPushButton:pressed
{	
	border-image: url(:/Skin/close.png)60 0 0 0 repeat
}
QPushButton:disabled
{	
	border-image: url(:/Skin/close.png)90 0 0 0 repeat
}
</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>172</y>
     <width>181</width>
     <height>87</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(131, 131, 131);
QPlainText{color: rgb(0, 0, 0);}</string>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>45</y>
     <width>248</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>案件名称:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditCaseName">
      <property name="text">
       <string>案件1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>监控点:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditCasePointName">
      <property name="text">
       <string>监控点1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>监控文件:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="BtnFileAdd">
      <property name="styleSheet">
       <string notr="true">QPushButton{
        background-color:rgb(6, 83, 131);
        border-width: 2px;
        border-color: rgb(150,150,150);
        border-style: solid;
        border-radius: 3;   
font-size:17px;   
color:rgb(255, 255, 255)
		}
QPushButton:hover{  border-color: rgb(160,160,160);background-color:rgb(10, 89, 150);;}
QPushButton:pressed{background-color:rgb(0, 83, 121);}
QPushButton:disabled{ background-color:rgb(110, 110, 110);;}</string>
      </property>
      <property name="text">
       <string>选择视频文件</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>268</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QPushButton" name="BtnOK">
      <property name="styleSheet">
       <string notr="true">QPushButton{
        background-color:rgb(6, 83, 131);
        border-width: 2px;
        border-color: rgb(150,150,150);
        border-style: solid;
        border-radius: 3;   
font-size:17px;   
color:rgb(255, 255, 255)
		}
QPushButton:hover{  border-color: rgb(160,160,160);background-color:rgb(10, 89, 150);;}
QPushButton:pressed{background-color:rgb(0, 83, 121);}
QPushButton:disabled{ background-color:rgb(110, 110, 110);;}</string>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="BtnCancel">
      <property name="styleSheet">
       <string notr="true">QPushButton{
        background-color:rgb(6, 83, 131);
        border-width: 2px;
        border-color: rgb(150,150,150);
        border-style: solid;
        border-radius: 3;   
font-size:17px;   
color:rgb(255, 255, 255)
		}
QPushButton:hover{  border-color: rgb(160,160,160);background-color:rgb(10, 89, 150);;}
QPushButton:pressed{background-color:rgb(0, 83, 121);}
QPushButton:disabled{ background-color:rgb(110, 110, 110);;}</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sisecplayer.qrc"/>
 </resources>
 <connections/>
</ui>
